backend/src/authentication/authentication.service.spec.ts

```diff
@@ -14,10 +14,11 @@ import { AdminModule } from '../admin/admin.module';
 import { AppModule } from '../app.module';
 import { RegisterAdminDto, LoginAdminDto } from '../admin/dto';
 import { BadRequestException, UnauthorizedException } from '@nestjs/common';
-import { Junior } from '../junior/junior.entity';
+import { Junior } from '../junior/entities/junior.entity';
 import { JuniorService } from '../junior/junior.service';
 import { JuniorModule } from '../junior/junior.module';
 import { RegisterJuniorDto, LoginJuniorDto } from '../junior/dto';
+import { Challenge } from '../junior/entities';
```

Comment:
can be imported with Junior => `{ Junior, Challenge } from './junior/entities';}`